GAE/src/org/waterforpeople/mapping/app/web/SurveyTaskServlet.java

```diff
@@ -77,37 +78,34 @@ protected RestResponse handleRequest(RestRequest req) throws Exception {
             if(s != null) {
                 surveyDao.delete(s);
             }
-        } else if (stReq.getAction().equals(
-                SurveyTaskRequest.DELETE_QUESTION_GROUP_ACTION)) {
+        } else if (stReq.getAction().equals(SurveyTaskRequest.DELETE_QUESTION_GROUP_ACTION)) {
             QuestionGroupDao qgDao = new QuestionGroupDao();
             QuestionGroup qg = qgDao.getByKey(id);
             if(qg != null) {
                 qgDao.delete(qg);
             }
-        } else if (stReq.getAction().equals(
-                SurveyTaskRequest.DELETE_QUESTION_ACTION)) {
+        } else if (stReq.getAction().equals(SurveyTaskRequest.DELETE_QUESTION_ACTION)) {
```

Comment:
2 things:
* wouldn't it make sense to extract stReq.getAction() to a local variable at the beginning? action = stReq.getAction();
* what if stReq.getAction() == null? I would use SurveyTaskRequest.DELETE_QUESTION_ACTION.equals(action)